Bottom: 1b72956cc36e0f283b956a5933c1af659ac00428
Top:    2106a3b6cb4189335cf36232fd7e4f1b8d71920d
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2008-12-30 10:47:46 -0500

Refresh of jds-soc-machine

---

diff --git a/sound/soc/fsl/soc-of-simple.c b/sound/soc/fsl/soc-of-simple.c
index 34a07e9..2d0ca0b 100644
--- a/sound/soc/fsl/soc-of-simple.c
+++ b/sound/soc/fsl/soc-of-simple.c
@@ -58,9 +58,9 @@ of_snd_soc_get_device(struct device_node *codec_node)
 	/* Initialize the structure and add it to the global list */
 	of_soc->codec_node = codec_node;
 	of_soc->id = of_snd_soc_next_index++;
-	of_soc->machine.dai_link = &of_soc->dai_link;
-	of_soc->machine.num_links = 1;
-	of_soc->device.machine = &of_soc->machine;
+	of_soc->card.dai_link = &of_soc->dai_link;
+	of_soc->card.num_links = 1;
+	of_soc->device.card = &of_soc->card;
 	of_soc->dai_link.ops = machine_ops;
 	list_add(&of_soc->list, &of_snd_soc_device_list);
 
@@ -158,8 +158,8 @@ int of_snd_soc_register_platform(struct snd_soc_platform *platform,
 
 	of_soc->platform_node = node;
 	of_soc->dai_link.cpu_dai = cpu_dai;
-	of_soc->device.platform = platform;
-	of_soc->machine.name = machine_name;
+	of_soc->card.platform = platform;
+	of_soc->card.name = machine_name;
 
 	/* Now try to register the SoC device */
 	of_snd_soc_register_device(of_soc);
@@ -173,13 +173,13 @@ EXPORT_SYMBOL_GPL(of_snd_soc_register_platform);
 void of_snd_soc_register_machine(char *name, struct snd_soc_ops *ops)
 {
 	struct of_snd_soc_device *of_soc;
-	
+
 	machine_name = name;
 	machine_ops = ops;
-	
+
 	list_for_each_entry(of_soc, &of_snd_soc_device_list, list) {
 		of_soc->dai_link.ops = machine_ops;
-		of_soc->machine.name = machine_name;
+		of_soc->card.name = machine_name;
 		of_snd_soc_register_device(of_soc);
 	}
